doc/Programming-guide/Key-platform-features/Endpoint-ownership/index.md

```diff
@@ -10,53 +10,61 @@ sort_idx: 40
-Then, Kaa SDK transfers this token to the Kaa cluster over a secure channel. The Kaa cluster verifies the access token and attaches the endpoint to the owner.
+## Using the owner access token
 
+As an owner, you can authenticate using an external authentication system and obtain your _access token_.
+You need to perform this authentication from the endpoint that is duly registered in the [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment).
+Your [endpoint SDK]({{root_url}}Glossary/#endpoint-sdk) sends this token to the [Kaa cluster]({{root_url}}Glossary/#kaa-cluster) over a secure channel.
+The Kaa cluster verifies the access token and attaches the endpoint to the owner.
 
 ![Owner access token flow](owner_attach_2.png "Owner access token flow")
 
+### Owner verifiers
 
-## Owner verifiers ##
-The owner verification is handled by specific server components called owner verifiers. 
-There are several default owner verifier implementations that are available out of the box for each Kaa installation. 
-This section contains general information about the architecture, configuration and administration of the default owner verifiers. 
-It is also possible to plug in [custom verifier implementations](#custom-owner-verifier). Each Kaa application can support multiple owner verifiers.
-You can add new verifier from Administration UI or with using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
-The following image example illustrates how to add new verifier from Administration UI.
+The owner verification is handled by specific server components called _owner verifiers_.
+There are several default implementations of the owner verifier available out of the box for each Kaa installation.
+This section contains general information about the architecture, configuration, and administration of the default owner verifiers.
+You can also plug in [custom verifier implementations](#custom-owner-verifier).
+One [Kaa application]({{root_url}}Glossary/#kaa-application) can support multiple owner verifiers.
+You can add a new verifier using the [Administration UI]({{root_url}}Glossary/#administration-ui) or [server REST API]({{root_url}}Programming-guide/Server-REST-APIs/#!/Verifiers/editUserVerifier).
+
+You can add user verifiers using the **User verifier** page of your application.
 
 ![new verifier](Admin-ui/adding-new-verifier.png "new verifier")
- 
-## Trustful owner verifier ##
-This owner verifier implementation is created for the test and debug purposes and always accepts provided owner id and access token. 
-It is recommended that you do not use this verifier in production because it may cause security issues. 
+
+### Trustful owner verifier
+
+This owner verifier implementation always accepts any owner ID and access token and is created for testing and debugging purposes.
 There is no specific configuration for this verifier, because its schema is empty.
```

Comment:
Doesn't explain. It's another way round. The schema is empty because we wanted no configuration.